utils/resources.go

```diff
@@ -57,11 +57,11 @@ var resourceRequirements = map[string]corev1.ResourceRequirements{
 	"sds": {
 		Limits: corev1.ResourceList{
 			"cpu":    resource.MustParse("2000m"),
-			"memory": resource.MustParse("7Gi"),
+			"memory": resource.MustParse("6Gi"),
```

Comment:
Any reason to decrease the memory limits?

Reply:
- ptal at https://issues.redhat.com/browse/ODFMS-152, it was discussed in great length and the repercussions are well aware
- however, we don't have a final say
- gist is, we want to accommodate 2 OSDs in a single m5.xlarge and 2 OSDs with 7Gi mem won't fit the bill